base/silent/src/main/java/org/openscience/cdk/silent/SilentChemObjectBuilder.java

```diff
@@ -143,7 +143,7 @@ private SilentChemObjectBuilder() {
         if (CDK_LEGACY_AC) {
             factory.register(IAtomContainer.class, AtomContainer.class);
         } else {
-            // System.err.println("[INFO] Using the new AtomContainer implementation.");
```

Comment:
I'll fix that... meant to switch it round. Changed the flag recently should now be... "Using Legacy AtomContainer"